src/modules/dashboard/components/ManageFundsDialog/ManageFundsDialog.tsx

```diff
@@ -177,7 +174,7 @@ const ManageFundsDialog = ({
       description: MSG.unlockTokensDescription,
       icon: 'emoji-padlock',
       onClick: () => callStep(nextStepUnlockToken),
-      permissionRequired: !canUnlockToken,
+      permissionRequired: !colony.canUnlockNativeToken,
```

Comment:
Just to confirm, we remove the option of raising the motion to unlock token?

Reply:
As far as I know, we haven't implemented the possibility of making a motion to unlock the token. There's even an issue in maintenance to update the modal to support that